equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/mapper/Crm14Mapper.java

```diff
@@ -3,11 +3,15 @@
 import org.apache.commons.lang3.StringUtils;
 import org.mapstruct.Mapper;
 import org.mapstruct.Mapping;
+import org.mapstruct.Named;
 import uk.gov.justice.laa.crime.equinity.historicaldata.generated.dto.*;
 import uk.gov.justice.laa.crime.equinity.historicaldata.model.*;
 
 @Mapper(componentModel = "spring")
 public interface Crm14Mapper extends CrmMapper {
+    final static String partnerInvolved_opt1="Co-defendant";
```

Comment:
`private static final String PARTNER_INVOLVED_OPT1 ="Co-defendant";`

string variable should be in Capital `PARTNER_INVOLVED_OPT1`

Reply:
done.